ouroboros-network/testlib/Test/Ouroboros/Network/Diffusion/Testnet/Cardano.hs

```diff
@@ -3015,16 +3019,16 @@ prop_diffusion_target_active_below ioSimTrace traceNumber =
        in counterexample
             ("\nSignal key: (local, established peers, active peers, " ++
              "recent failures, opportunities, is node running, ignored too long)") $
-          counterexample
-            (List.intercalate "\n" $ map show $ Signal.eventsToList events) $
+          -- counterexample
+          --   (List.intercalate "\n" $ map show $ Signal.eventsToList events) $
```

Comment:
Are you sure we want to comment out this? If we do, we should also comment out the previous one as well.